openlibrary/templates/lists/home.html

```diff
@@ -41,7 +41,7 @@ <h2>$_('Active Lists')</h2>
                     <a href="$list.key" class="title results">$:_('%(listname)s </a> from <a href="%(key)s">%(owner)s</a>', key=owner.key, listname=list.name, owner=owner.displayname)
                 </div>
                 <div class="meta">
-                    $ msg = ungettext("1 seed", "%(count)d seeds", len(list.seeds))
+                    $ msg = ungettext("1 Book", "%(count)s Books", len(list.seeds))
```

Comment:
Is the change from %(count)d to %(count)s intentional?

Reply:
Good catch. That's wrong. The same value should be used both places, presumably `len(list.seeds)` since that's what's being used to conditionalize the pluralization (although I didn't go check the code).

Reply:
![image](https://user-images.githubusercontent.com/64412143/104105425-fc55f180-52d3-11eb-9172-b0ecf0932369.png)
these numbers uses %s so I wrote %s.
you want it to be an integer(%d)?

Reply:
```suggestion
                    $ msg = ungettext("1 item", "%(count)d items", len(list.seeds))
```